src/packagedcode/cargo.py

```diff
@@ -179,3 +193,29 @@ def parse_person(person):
 person_parser_no_name = re.compile(
     r'(?P<email><([^>]+)>)?'
 ).match
+
+
+def build_cargo_lock_package(package_data):
+    """
+    Return a Package object from a Cargo.lock package data mapping or None.
+    """
+
+    package_dependencies = []
+    core_package_data = package_data.get('package', {})
```

Comment:
is this a mapping or a list?

Reply:
Ah...It's a list. I will change it soon.